tstyche.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "testFileMatch": ["**/*.spec.ts"]
```

Comment:
It will type check all files matching the pattern and will run any type test if present.